scripts/workers/code_upload_worker_utils/monitor_submission.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/sh
+echo "Monitoring submission..."
+time_elapsed=0
+while [ $time_elapsed -le $SUBMISSION_TIME_LIMIT ]
+do
+    echo "Checking for submission file after $time_elapsed secs.."
+    sh /evalai_scripts/make_submission.sh
+    if [ -f "$SUBMISSION_PATH/completed.txt" ]
+    then
+        echo "Submission Complete, Exiting."
+        exit 0
+    fi
+    if [ $time_elapsed -lt $SUBMISSION_TIME_LIMIT -a $(( $time_elapsed + $SUBMISSION_TIME_DELTA )) -gt $SUBMISSION_TIME_LIMIT ]
+    then
+        next_checkpoint=$SUBMISSION_TIME_LIMIT
+    else
+        next_checkpoint=$(( $time_elapsed + $SUBMISSION_TIME_DELTA ))
+    fi
+    if [ $next_checkpoint -le $SUBMISSION_TIME_LIMIT ]
+    then
+        sleep $(( $next_checkpoint - $time_elapsed ))
+    fi
+    time_elapsed=$next_checkpoint
+done
+url="$EVALAI_API_SERVER/api/jobs/challenge/$CHALLENGE_PK/update_submission/"
+submission_data="{\
+    \"challenge_phase\": $PHASE_PK,\
+    \"submission\": $SUBMISSION_PK,\
+    \"stdout\": \"\",\
+    \"stderr\": \"submission.json/submission.csv not found.\",\
+    \"submission_status\": \"failed\",\
+    \"result\": \"[]\",\
+    \"metadata\": \"\"\
+}"
+curl_request="curl --location --request PUT '$url' -H 'Content-Type: application/json' --header 'Authorization: Bearer $AUTH_TOKEN' -d '$submission_data'"
+eval $curl_request
+echo "\nFile submission failed."
```

Comment:
This error is misleading right? This block is executed when submissions times out correct? If that is the case please make `stderr: "Execution time limit exceeded"`. 

We should handle error messages correctly here. If the submission containers exits before creating `submission.csv/json` show `submission.csv not found` otherwise if container is still running after n hours we should `Execution time limit exceeded`.

Reply:
Yeah, we have a plan to check the health of submission docker from sidecar in near future, would add this error there. Changing this error to TLE error as suggested.

Reply:
Updated.